### ISSUE TITLE
Support having path level parameters

### ISSUE THREAD

**Issue by jszwedko** [westfieldlabs/apivore#59]
Currently apivore expects everything under a given route, e.g. `/posts`, to be a HTTP verb, but `parameters` is also supported (to specify parameters that are required by each verb under the path).

E.g.

```yml
...
paths:
  /posts/{id}:
    parameters:
      - name: id
        in: path
        type: integer
        required: true
    get:
      ...
    put:
      ...
...
```

**Comment by jszwedko** [comment]
See https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#path-item-object

**Comment by hornc** [comment]
Thanks for the reference back to the swagger-spec,  looks good. I have merged and added an example to  the sample swagger.json for apivore's tests https://github.com/westfieldlabs/apivore/pull/61

**Comment by hornc** [comment]
Released in Apivore 1.4.0
